terraform_files/nutanix-ci-machine/variables-nutanix.tf

```diff
@@ -57,10 +51,32 @@ variable "vcpu" {
 
 variable "disk_size" {
   type        = number
-  default     = 240
+  default     = 650
```

Comment:
Why is it so big? I think that 240 was too much

Reply:
I was too but it seems that the test infra needs a lot more 

log from run before this change
```
{  age nmstate-devel-2.2.35-1.el9.x86_64 needs 410MB more space on the / filesystem
  installing package pipewire-jack-audio-connection-kit-1.0.1-1.el9.x86_64 needs 410MB more space on the / filesystem
  installing package pipewire-alsa-1.0.1-1.el9.x86_64 needs 410MB more space on the / filesystem
  installing package pipewire-pulseaudio-1.0.1-1.el9.x86_64 needs 411MB more space on the / filesystem
  installing package libvirt-client-10.5.0-5.el9.x86_64 needs 412MB more space on the / filesystem
  installing package libvirt-devel-10.5.0-5.el9.x86_64 needs 414MB more space on the / filesystem
  installing package rpm-plugin-selinux-4.16.1.3-34.el9.x86_64 needs 414MB more space on the / filesystem
  installing package gcc-11.5.0-2.el9.x86_64 needs 493MB more space on the / filesystem
  installing package openssl-devel-1:3.2.2-6.el9.x86_64 needs 527MB more space on the / filesystem
  installing package xorriso-1.5.4-4.el9.x86_64 needs 528MB more space on the / filesystem
  installing package postgresql-13.16-1.el9.x86_64 needs 531MB more space on the / filesystem
  installing package sqlite-3.34.1-7.el9.x86_64 needs 533MB more space on the / filesystem

Error Summary
-------------
Disk Requirements:
   At least 533MB more space needed on the / filesystem.
  ```

Reply:
The description says that this field is in GB, the error message is in MB.
Am I missing somehing?

Reply:
yeah I wiill  update it you correct

Reply:
I've checked - this is reported  in GB in the var and converted to bytes in the terrraform file